### ISSUE TITLE
version 8.4.0 : Cannot find module 'compression'

### ISSUE THREAD

**Issue by rlebosse** [HenrikJoreteg/hjs-webpack#291]
I updated my hjs-webpack version from **8.3.0** to **8.4.0** and I have the following error when trying to start my app :

```sh
> my-app@0.1.1 start /home/rle/app/frontoffice
> cross-env NODE_ENV=dev PORT=8080 EXTERNAL=1 BROWSERSLIST_CONFIG=./browserslist ./node_modules/.bin/hjs-dev-server

module.js:457
    throw err;
    ^

Error: Cannot find module 'compression'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/rle/app/frontoffice/node_modules/hjs-webpack/bin/hjs-dev-server.js:11:16)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)

npm ERR! Linux 4.4.0-34-generic
npm ERR! argv "/usr/local/bin/node" "/home/rle/npm-global/bin/npm" "start"
npm ERR! node v6.7.0
npm ERR! npm  v3.10.1
npm ERR! code ELIFECYCLE
npm ERR! my-app@0.1.1 start: `cross-env NODE_ENV=dev PORT=8080 EXTERNAL=1 BROWSERSLIST_CONFIG=./browserslist ./node_modules/.bin/hjs-dev-server`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the my-app@0.1.1 start script 'cross-env NODE_ENV=dev PORT=8080 EXTERNAL=1 BROWSERSLIST_CONFIG=./browserslist ./node_modules/.bin/hjs-dev-server'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the my-app package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cross-env NODE_ENV=dev PORT=8080 EXTERNAL=1 BROWSERSLIST_CONFIG=./browserslist ./node_modules/.bin/hjs-dev-server
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs my-app
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls my-app
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/rle/app/frontoffice/npm-debug.log
```

**Comment by lukekarrys** [comment]
Sorry about that! I just released `8.4.1` which should fix this. Thanks for the report!

**Comment by rlebosse** [comment]
Works like a charm, thanks!
